deis-tests/manifests/deis-tests-pod.yaml

```diff
@@ -9,17 +9,13 @@ spec:
   restartPolicy: Never
   containers:
   - name: deis-e2e
-    image: quay.io/deisci/deis-e2e:canary
+    image: quay.io/deisci/deis-e2e:git-7f7d775
```

Comment:
Shouldn't this chart always be referencing the `:canary` tag though? If the tests themselves are broken, we should fix the tests so that we get a new canary image. Sorry, I must be missing some detail here.